packages/expo-permissions/android/src/main/java/expo/modules/permissions/requesters/LocationRequester.kt

```diff
@@ -40,10 +54,20 @@ class LocationRequester : PermissionRequester {
           PermissionsStatus.UNDETERMINED.status
         }
       })
+      scope =
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+          val accessBackgroundLocation = permissionsResponse.getValue(Manifest.permission.ACCESS_BACKGROUND_LOCATION)
+          if (accessBackgroundLocation.status == PermissionsStatus.GRANTED) SCOPE_ALWAYS else SCOPE_IN_USE
```

Comment:
```suggestion
          if (accessBackgroundLocation.status == PermissionsStatus.GRANTED) {
            SCOPE_ALWAYS 
          } else { 
            SCOPE_IN_USE
          }
```
With brackets, it's more readable ;) cause you can see what returns the `when` statement.